generator/views/view.py

```diff
@@ -8,11 +8,12 @@
 OMIT_VIEWS: Set[str] = set()
 
 
-class ViewDict(TypedDict):
+class ViewDict(TypedDict, total=False):
```

Comment:
Marking all keys as non-required isn't ideal, but I couldn't figure out another way to satisfy mypy's type checks at the moment.

In Python 3.11 `TypedDict` supports marking individual keys as non-required, so whenever we upgrade we can adjust this.

Reply:
this should be a comment in the file, so we know to fix it later.